docs/en/guides/developer/lightweight-update.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: /en/guides/developer/lightweght-update
+sidebar_label: Lightweight Update
+title: Lightweight Update
+keywords: [lightweight update]
+---
+
+## On-fly mutations
+
+On-fly mutations can be enabled for `MergeTree`-family tables by query-level setting `apply_mutations_on_fly`. On-fly mutations help to achieve the behaviour when you see the effect of `UPDATE` or `DELETE` queries on `SELECT`-s immediately after the `UPDATE` or `DELETE` query is finished. With enabled `apply_mutations_on_fly` all mutations that are not materialized but should be applied for data part will be applied during `SELECT` queries. Only `UPDATE` and `DELETE` mutations can be applied on-fly.
```

Comment:
> On-fly mutations can be enabled

On-fly mutation is an implementation of the lightweight update feature, it can be enabled